builder/query_test.go

```diff
@@ -105,6 +105,22 @@ func TestQuery_Build(t *testing.T) {
 			result: "SELECT `users`.* FROM `users` FOR UPDATE;",
 			query:  rel.From("users").Lock("FOR UPDATE"),
 		},
+		{
+			result: "SELECT `c`.`id`,`c`.`name` FROM `contacts` `c`;",
+			query:  rel.Select("c.id", "c.name").From("contacts c"),
+		},
+		{
+			result: "SELECT `c`.`id`,`c`.`name` FROM `contacts` AS `c`;",
+			query:  rel.Select("c.id", "c.name").From("contacts as c"),
+		},
+		{
+			result: "SELECT `c`.`id`,`c`.`name` FROM `contacts` `c`;",
+			query:  rel.Select("c.id").Select("c.name").From("contacts c"),
```

Comment:
I wonder why not using single Select function? seems this change is also not related to the alias fix?

Reply:
I see, then can you create separate PR for this?